src/google-apis/auth-client.ts

```diff
@@ -62,9 +57,10 @@ var API = 'https://clouderrorreporting.googleapis.com/v1beta1';
  * @property {Object} _logger - the instance-cached logger instance
  */
 export class RequestHandler extends common.Service {
-  private _config: any;
-  private _logger: any;
-  private authClient: any;
+  private _config: Configuration;
```

Comment:
Feels like we're inconsistent with leading underscores on private properties.

Reply:
I agree, but for these PRs, I didn't want to make any changes to the code that wasn't required for the translation to Typescript.  There will be a PR coming that enables `gts check` and contains a run of `gts fix`.  That might address these inconsistencies too.